Automattic/wp-calypso
# Anchor: Customize welcome modal for Anchor users

#### Changes proposed in this Pull Request

Customizes the welcome guide displayed in the block editor for Anchor users who have signed up to WP.com.

<img width="752" alt="Screen Shot 2020-12-15 at 16 16 53" src="https://user-images.githubusercontent.com/1233880/102233953-15e06500-3ef1-11eb-864e-3fea05fe8d6b.png">

#### Testing instructions

- Generate a new build of the Editing Toolkit plugin and sync it to your sandbox: `cd apps/editing-toolkit && yarn dev --sync`.
- Sandbox a test site.
- Go to https://wordpress.com/post and select the test site you sandboxed above.
- Click on Options (aka "3 dots") and select "Welcome guide".
- Make sure the normal Welcome guide for users is displayed (first step: "Welcome to your website").
- Mark your site as a podcasting site:
  - Open a WP shell in your WP.com sandbox: `wp shell --user=<YOUR_WPCOM_USERNAME> --url=<YOUR_SITE_ADDRESS>`.
  - Require the Anchor lib: `require_once( WP_CONTENT_DIR . '/lib/anchor.php' );`
  - Update the site : `\A8C\Anchor\mark_site_as_podcasting_site( <SITE_ID>, 'test_podcast_id' );`.
- Add the Anchor params to the editor URL (e.g. `?anchor_podcast=22b6608&anchor_episode=e324a06c-3148-43a4-85d834c0d8222138&spotify_show_url=https%3A%2F%2Fopen.spotify.com%2Fshow%2F6HTZdaDHjqXKDE4acYffoD%3Fsi%3DEVfDYETjQCu7pasVG5D73Q`).
- Open the "Welcome guide" again.
- Make sure the custom Welcome guide for Anchor is displayed (first step: "Create your first episode").

Fixes 323-gh-Automattic/dotcom-manage
